Add solutions to lab 4.1 (#210)
acend/openshift-4-ops-training: content/en/docs/04/01_basics.md

The command will error out if you don't write it.
{{% /alert %}}

{{% /details %}}

> After a quick moment, you are presented with a root shell on the desired node.

Reviewer: ```suggestion
After a brief moment you are presented with a root shell on the desired node.
```

However, you're not yet really on the node, you're limited to what the container sees of it.

> You need to chroot onto the node first before you'll be able to, e.g., use host binaries. The message that appeared when you opened the debug shell conveniently provides the chroot command for us.

Reviewer: ```suggestion
You need to chroot onto the node before you'll be able to use host binaries. The message that appeared when you opened the debug shell conveniently provides the chroot command for us.
```

{{% details title="Hints" mode-switcher="normalexpertmode" %}}

```bash
chroot /host
```

{{% /details %}}

> If you're not sure if you're on the node, there are different possibilities to find out, one of which is looking at the release file.

Reviewer: ```suggestion
If you're not sure whether you're on the node, there are different possibilities to find out, one of which is looking at the release file.
```

If the output says `Red Hat Enterprise Linux release [...]`, you're still contained in the container.
As we know, OpenShift nodes use CoreOS as operating system, so the output should read `Red Hat Enterprise Linux CoreOS release [...]`.

{{% /details %}}

We can now do all the things as if we had connected via ssh.

> Which, of course, is still possible but might be less convenient.

Reviewer: ```suggestion
Which is still possible but might be less convenient.
```

`oc adm top images` is your friend.
It works very similarly for `ImageStreams` objects.

> What gives you a nice overview of resource usage on your nodes (without using Prometheus) is `oc adm top node`, which also works with the `--selector` parameter to list all nodes that match the selected label:

Reviewer: ```suggestion
A command providing you with a nice overview of resource usage on your nodes (without using Prometheus) is `oc adm top node`, which also works with the `--selector` parameter to list all nodes that match the selected label:
```
